mixnet/node.py

```diff
@@ -1,107 +1,186 @@
 from __future__ import annotations
 
 import asyncio
-from contextlib import suppress
-from typing import Self, Tuple, TypeAlias
+import hashlib
+from typing import Awaitable, Callable, TypeAlias
 
-from cryptography.hazmat.primitives.asymmetric.x25519 import (
-    X25519PrivateKey,
-)
+from pysphinx.payload import DEFAULT_PAYLOAD_SIZE
 from pysphinx.sphinx import (
     Payload,
     ProcessedFinalHopPacket,
     ProcessedForwardHopPacket,
     SphinxPacket,
-    UnknownHeaderTypeError,
-)
-
-from mixnet.config import MixNodeConfig, NodeAddress
-from mixnet.poisson import poisson_interval_sec
-
-PacketQueue: TypeAlias = "asyncio.Queue[Tuple[NodeAddress, SphinxPacket]]"
-PacketPayloadQueue: TypeAlias = (
-    "asyncio.Queue[Tuple[NodeAddress, SphinxPacket | Payload]]"
 )
 
+from mixnet.config import GlobalConfig, NodeConfig
+from mixnet.packet import Fragment, MessageFlag, MessageReconstructor, PacketBuilder
 
-class MixNode:
-    """
-    A class handling incoming packets with delays
+NetworkPacket: TypeAlias = "SphinxPacket | bytes"
+NetworkPacketQueue: TypeAlias = "asyncio.Queue[NetworkPacket]"
+Connection: TypeAlias = NetworkPacketQueue
+BroadcastChannel: TypeAlias = "asyncio.Queue[bytes]"
```

Comment:
Why the types are stringified (`"`)? Just curious here

Reply:
Removed `"`s. I added that for some reason previously but not needed anymore: 235c703